Directory.Build.targets

```diff
@@ -26,7 +26,7 @@
     When .NET gets built from source, make the SDK aware there are bootstrap packages
     for Microsoft.NETCore.App.Runtime.<rid> and Microsoft.NETCore.App.Crossgen2.<rid>.
   -->
-  <ItemGroup Condition="'$(DotNetBuildFromSource)' == 'true'">
+  <ItemGroup Condition="'$(IgnoreMibc)' == 'true'">
```

Comment:
Probably not here?

Reply:
No, I mean this explicitly checks SV and not mibc, should keep checking SB flag

Reply:
Ah right, that was by accident, reverted.